ios-target/Extensions/UIButtonExtension.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  UIButtonExtension.swift
+//  ios-target
+//
+//  Created by sol manini on 7/1/19.
+//  Copyright © 2019 TopTier labs. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+extension UIButton {
+  
+  func enableButton(enable: Bool) {
```

Comment:
since you already have `enable` in the function name I would loose the parameter alias: `func enableButton(_ enable: Bool)`